package p2;


public static void main(String args[]) 
{   
    try { 
        fun(); 
    } 
    catch (Exception e) { 
        System.out.print("caught in main."); 
    }
    System.out.print("All is well"); 
} 
} 
       